unix-domain-socket/src/test/scala/docs/scaladsl/UnixDomainSocketSpec.scala

```diff
@@ -31,52 +31,71 @@ class UnixDomainSocketSpec
 
   implicit val ma: ActorMaterializer = ActorMaterializer()
 
+  private val dir = Files.createTempDirectory("UnixDomainSocketSpec")
+
   "A Unix Domain Socket" should {
-    "receive what is sent" in {
+    "receive some bytes" in {
       //#binding
       val file: java.io.File = // ...
         //#binding
-        Files.createTempFile("UnixDomainSocketSpec1", ".sock").toFile
-      file.delete()
-      file.deleteOnExit()
+        dir.resolve("sock1").toFile
+
+      val received = Promise[ByteString]
+
+      val serverSideFlow = Flow[ByteString]
+        .buffer(1, OverflowStrategy.backpressure)
+        .wireTap(bytes => received.success(bytes))
```

Comment:
The details of the flow itself are unimportant to the doc reader. I also changed the nature of the flow here so that it will not close downstream if there's still something to be sent.